### ISSUE TITLE
ERROR in Path must be a string. Received undefined

### ISSUE THREAD

**Issue by ghost** [splunk/parallel-piper#3]
Im trying to use Splunk parallel-piper, and followed all the docuemntation, however when I build my Node.Js, I receive an error message that Im not able to solve it.

Any guidance is appreciated:

```
    sh build.sh
```

 43% 23/41 build modulesThe unused option is deprecated. Please use options.discardUnused instead.
The urls option is deprecated. Please use options.normalizeUrl instead.
The idents option is deprecated. Please use options.mergeIdents instead.
The idents option is deprecated. Please use options.reduceIdents instead.
Hash: c39b586fc444f14cb888  
Version: webpack 1.13.1
Time: 3855ms
                                Asset      Size  Chunks             Chunk Names
 9cbe61a2b483128aadcd2d38d9a51fa6.svg   36.7 kB          [emitted]  
 d39c7c45536d58b6d370d09b74b37bfa.eot   11.7 kB          [emitted]  
 3b777c30a2fc8ad75220c4b86556c5eb.ttf   21.9 kB          [emitted]  
23ed833c737fb20e3138ec99fd1db0df.woff   13.5 kB          [emitted]  
                             piper.js    131 kB       0  [emitted]  main
                            piper.css   7.41 kB       0  [emitted]  main
                           index.html  52 bytes          [emitted]  
    + 154 hidden modules

WARNING in piper.js from UglifyJs
Condition always true [./~/jquery/dist/jquery.js:9778,0]
Dropping side-effect-free statement [./~/virtual-dom/virtual-hyperscript/index.js:123,0]

ERROR in Path must be a string. Received undefined`

**Comment by YannickNoPanic** [comment]
If anyone finds this and is still dealing with this:
https://github.com/webpack-contrib/css-loader/issues/126 this is basically the same
https://github.com/webpack-contrib/css-loader/commit/3d2435cb69eb63cf7dedea12b8d36401d33a580c this is the solution

**Comment by ohmyguigs** [comment]
I made a workaround like this:
https://github.com/vtex/splunk-meetup/commit/e0ec9118ddd9cd0998fc88c3031fb90cc22843eb

tldr;
Adding `publicPath: ''` in output of webpack.config
